crates/wit-component/tests/interfaces/console.wat

```diff
@@ -11,7 +11,7 @@
     )
   )
   (export (;1;) "console" (type 0))
-  (@custom "package-docs" "\00{}")
+  (@custom "package-docs" "\01{}")
```

Comment:
Why does this get bumped to 01?  I don't really see reasoning in this PR but I assume it is due to the face this feature is no turned on?

Reply:
Ah that's [around here](https://github.com/bytecodealliance/wasm-tools/blob/84d3a4ce8328925e5691877cbf9084ddcab26caa/crates/wit-parser/src/metadata.rs#L121-L133) where by changing the static this is changing the default emission as well